x-pack/plugin/esql/qa/testFixtures/src/main/resources/show.csv-spec

```diff
@@ -9,176 +9,176 @@ v:long
-to_ul                    |"unsigned_long to_ul(v:boolean|date|keyword|text|double|long|unsigned_long|integer)"             |v   |"boolean|date|keyword|text|double|long|unsigned_long|integer"                                              |                                                    |unsigned_long                    |                                                                                       |false                       |false          
-to_ulong                 |"unsigned_long to_ulong(v:boolean|date|keyword|text|double|long|unsigned_long|integer)"          |v   |"boolean|date|keyword|text|double|long|unsigned_long|integer"                                              |                                                    |unsigned_long                    |                                                                                       |false                       |false          
-to_unsigned_long         |"unsigned_long to_unsigned_long(v:boolean|date|keyword|text|double|long|unsigned_long|integer)"  |v   |"boolean|date|keyword|text|double|long|unsigned_long|integer"                                              |                                                    |unsigned_long                    |                                                                                       |false                       |false          
-to_ver                   |"version to_ver(v:keyword|text|version)"                                                         |v   |"keyword|text|version"                                                                                     |                                                    |version                          |                                                                                       |false                       |false          
-to_version               |"version to_version(v:keyword|text|version)"                                                     |v   |"keyword|text|version"                                                                                     |                                                    |version                          |                                                                                       |false                       |false          
-trim                     |"keyword|text trim(str:keyword|text)"                   |str                      |"keyword|text"    | ""                                                 |"keyword|text"       |Removes leading and trailing whitespaces from a string.| false | false
+       name:keyword      |                        synopsis:keyword          |       argNames:keyword  | argTypes:keyword |             argDescriptions:keyword                |returnType:keyword   |    description:keyword  | optionalArgs:boolean | variadic:boolean | isAggregation:boolean               
```

Comment:
I'm sorry for how this test ends up looking. It's quite a lot.

Reply:
I was surprised to see that there's no update task for this kind of snapshot tests. I think it would be feasible to create one.

Reply:
Yeah. I do these one at a time so I just manage it by hand. It isn't like this blob is really code reviewable either.

I do think we should do *something* about this. Or at least think about it.